.gitignore

```diff
@@ -26,6 +26,8 @@ vendor
 logs
 *.log
 *DS_Store
+*.env
```

Comment:
Actually, I think it's okay not to include `.env` to `.gitignore` so collaborators or maintainers will have an idea as to what env variables are necessary. Each person could then have a `.env.development` or `.env.test` locally on their own machines, and these are the files that will be included in `.gitignore`

Reply:
Sure thing.